assets/css/translation.css

```diff
@@ -126,3 +126,116 @@ html[dir='rtl'] #srcLanguages {
     padding: 0 15px;
     position: relative;
 }
+
+#apertiumLogo {
+    margin-bottom: -.25em;
+}
+
+#applyingWidth {
+    width: 35%;
+}
+
+#applyingPadding {
```

Comment:
I'm afraid this `id` isn't too descriptive either since it isn't related to the actual HTML element. Do you have any other ideas?

Reply:
for width i would say id: applyingPullRightandHidden
for padding i would say id: applyingLocaleGlobe
any suggestios on  these names?

Reply:
Rule of thumb: elements are things (nouns) and ids should also be like nouns.

Reply:
Hi sushain as u said I was thinking of summarizing what that (applyingPullRightHidden) Div is for and for knowing purpose I commented the whole div out and built using make file. I couldn't find any notable glitches!! So stuck in naming this

Reply:
I can't find `applyingPullRightHidden` in your code anymore so I don't know what div that is.